qemu/tests/blockdev_commit_fio.py

```diff
@@ -0,0 +1,65 @@
+from virttest import utils_test
+from provider import job_utils
+from provider import backup_utils
+import logging
+from threading import Thread
+import random
+import time
+
+from provider.blockdev_commit_base import BlockDevCommitTest
+from provider.storage_benchmark import generate_instance
```

Comment:
1. From code_style in tp-qemu:

The order of imports should be as follows:
 
 Standard python modules
 Non-standard python modules
 Autotest modules
2. Thread is not used here?

Reply:
Have updated, thanks zhenchao.